### PR TITLE
1412 r ops UI improve readability of event history

### PR DESCRIPTION
**Pull request by ryangrundy7**
# Motivation and Context

To help view case event history on the r ops ui case details page, the events are now in collapsible boxes which can be expanded to view more details for that specific case event.

# What has changed

- Case events are now in collapsible boxes

# How to test?

- Run the ATs against the branch.
- View the details of some cases and the case events should be in collapsible boxes which can be expanded to view more information.

# Links

[Trello](https://trello.com/c/hvudIE4T/)

### CODE REVIEW

**Comment by ryangrundy7** [comment]
[R Ops UI - Improve readability of event history (8)](https://trello.com/c/hvudIE4T/1412-r-ops-ui-improve-readability-of-event-history-8)

**Review by liamtoozer — APPROVED** [review]
Looks great!

**Review by gavinleeedwards — APPROVED** [review]
All looks good.

File: census_rm_ops_ui/iap_audit.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ def log_iap_audit(iap_audience):
     if not iap_jwt:
         return
     g.user = iap_jwt['email']
-    logger.info(audit=True,
+    logger.info('Logging iap audit', audit=True,
```
Comment (AdamHawtin) [inline census_rm_ops_ui/iap_audit.py]: Why this change?

Comment (ryangrundy7) [reply census_rm_ops_ui/iap_audit.py]: Removed now as I've pinned the packages that did it. They were doing some major upgrades in versions so I've pinned them for now
